build/main.rs

```diff
@@ -52,6 +52,9 @@ fn main() {
         println!("Failed to run rustfmt, will skip formatting generated files.")
     }
 
+    println!("cargo:rerun-if-changed=.cbindgen.toml");
+    generate_c_bindings();
```

Comment:
I think it might make sense to move the `println!` into `generate_c_bindings`, so that everything is in ine place.